### ISSUE TITLE
Display currently installed version

### ISSUE THREAD

**Issue by anntzer** [pbrisbin/downgrade#64]
Just a minor usability enhancement suggestion: display the currently installed version, e.g.
```
Currently installed: x.y.z

Available packages:

   ...
```
or
```
Available packages:

   1) ... (local, currently installed)
   ...
```
(although the second case would fail to cover the case where the currently installed version is locally built and not remotely available).

**Comment by pbrisbin** [comment]
FWIW there's an unreleased feature on master right now that places a `*` next to entries that have been previously installed. Not quite the same (it doesn't highlight *currently* installed, just any that were at any point installed). Maybe this is good enough for your needs?

**Comment by pbrisbin** [comment]
This was closed by #66
